tests/common/helpers/drop_counters/drop_counters.py

```diff
@@ -95,7 +95,12 @@ def get_drops_across_all_duthosts():
         return drop_list
     check_drops_on_dut = lambda: packets_count in get_drops_across_all_duthosts()
     if not wait_until(25, 1, check_drops_on_dut):
-        fail_msg = "'{}' drop counter was not incremented on iface {}. DUT {} == {}; Sent == {}".format(
-            column_key, dut_iface, column_key, get_drops_across_all_duthosts(), packets_count
-        )
-        pytest.fail(fail_msg)
+        # The actual Drop count should always be equal or 1 or 2 packets more than what is expected due to some other drop may occur
+        # over the interface being examined. When that happens if looking onlyu for exact count it will be a false positive failure.
+        # So do one more check to allow up to 2 packets more dropped than what was expected as an allowed case.
+        if ((packets_count+2) in get_drops_across_all_duthosts()) or ((packets_count+1) in get_drops_across_all_duthosts()):
```

Comment:
seems like this method will be called multiple times, could you cache its result somewhere and reuse it?

Reply:
@Blueve Are you saying that you like to cache the result of "get_drops_across_all_duthosts()" to a local list and use that for comparison purpose?

Reply:
Cache the return value of ```get_drops_across_all_duthosts``` is a good idea.

Reply:
Agreed.  Done!